packages/calling/src/CallingClient/calling/call.ts

```diff
@@ -2416,10 +2434,13 @@ export class Call extends Eventing<CallEventTypes> implements ICall {
             break;
           }
 
-          case RoapScenario.ANSWER:
+          case RoapScenario.ANSWER: {
```

Comment:
Update the corresponding test file to cover unit test for this change

Reply:
We had a discussion about this in the Scrum meeting this morning and Sreekanth ruled out that it is a string manipulation and we need not write UTs for this.

I saw for the `RoapScenario.OFFER` case also. The test file doesn't check for the manipulation

Reply:
@mkesavan13 I was under the impression that the manipulation of the SDP was happening within an existing function which is already being tested. Since the changeset is actually a new function, let's write a UT for this.